src/index.html

```diff
@@ -20,23 +20,23 @@ <h2>Recent Scores</h2>
             </div>
             <div class="score-table">
                 <ul class="scorelist">
-                    <li class="score">James: 100</li>
+                    <!-- <li class="score">James: 100</li>
                     <li class="score">Tim: 200</li>
                     <li class="score">David: 120</li>
                     <li class="score">Henry: 211</li>
                     <li class="score">John: 210</li>
                     <li class="score">Thomas: 100</li>
-                    <li class="score">Gerald: 100</li>
+                    <li class="score">Gerald: 100</li> -->
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.